### DISCUSSION TITLE
How to limit ingest data only if client provide an API key only?

### DISCUSSION THREAD

**Discussion by doggy8088** [datalust/seq-tickets#1557] [Help] [answered]
I tried to not providing an API from my client, but Seq still ingest logs from that client. Can I limit access to all clients that have valid API key to send logs to Seq?

**Comment by nblumhardt** [comment, marked as answer]
Hi Will! Yes, on the _Settings > API keys_ screen, you should find a checkbox "Require an API key for HTTP/S ingestion". If you tick that box, only clients with valid API keys will be able to send logs.

Does this do what you're after?

Best regards,
Nick

**Reply by doggy8088** [reply]
To be honest, I misread the `Require authentication for HTTP/S ingestion` as `Require HTTP/S for ingestion`. My bad. 😅

**Reply by doggy8088** [reply]
Thanks for you answer! 👍
